CardanoSharp.Wallet/Models/Transactions/TransactionUnspentOutput.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace CardanoSharp.Wallet.Models.Transactions
+{
+    public class TransactionUnspentOutput
+    {
+        public List<TransactionInput> Inputs { get; set; }
```

Comment:
sorry guys, i dont understand why a utxo has multiple inputs and outputs, dont know if i missed a convo or if im misunderstanding what its meant to represent, but i thought a utxo was a single output from a single input?

Reply:
> sorry guys, i dont understand why a utxo has multiple inputs and outputs

Yeah it has been turned more into a `CoinSelectionResponse`

The List of Outputs is really "change" in this case
The List of Inputs is the list of inputs required for the transaction.

That being said representing a UTxO represented as a model could be how we described. imo, its more about how you would like to view the relationship. all a UTxO does is point to a previous transaction of which it was an output. It could definitely be represented as a single output as you described, but i wouldn't claim the single input part. if i send you 100 ADA, and have a 40 ADA UTxO and a 80 ADA UTxO, i will need both as input to send you the single output of 100 ADA.

Reply:
to be more clear, I think having a model called TransactionUnspentOutput should be a single output and a list of inputs.

Reply:
right... yeah that makes sense. im thinking of the overlap here with the work im doing for blazor and the cip30 stuff. utxo class should be standardized across them i reckon, otherwise we going to be duplicating models etc.
i havent seen any utxos from wallets yet with more than 1 input, but that might just be my testing data not being extensive enough... tbh i hadnt thought of it as possibly having more than 1 input as the cip30 spec has it as:

transaction_unspent_output = [
input: transaction_input,
output: transaction_output,
]

Reply:
did some exploring on how others define the UTxO. i could find anything in the rust sdk, but koios seems to have a good definition. 

```json
"utxo_set": [
  {
      "tx_hash": "f144a8264acf4bdfe2e1241170969c930d64ab6b0996a4a45237b623f1dd670e",
      "tx_index": 0,
      "block_height": 42325043,
      "block_time": "2022-02-15T18:42:13",
      "value": 157832856,
      "datum_hash": "5a595ce795815e81d22a1a522cf3987d546dc5bb016de61b002edd63a5413ec4",
      "asset_list": [
      {
         "policy_id": "6cf6b5cf0fefbe9e69d640d8be84912bb2c9e132671954548790bcfb",
         "asset_name": "6d65736d6572697a65723038353436",
         "quantity": 1
      }
   }
]
```

I definitely like this definition. a flattened view of assets is always helpful.

Reply:
cool cool... bit of a deviation from current style and structure, but conceptually same same mostly, just the policy id not being used as a key